src/System.Private.CoreLib/shared/System/BitOps.cs

```diff
@@ -13,6 +13,12 @@
 
 namespace System
 {
+    /// <summary>
+    /// Utility methods for intrinsic bit-twiddling operations.
+    /// The methods use hardware intrinsics when available on the underlying platform,
+    /// otherwise they use optimized software fallbacks.
+    /// Operations are all little-endian.
```

Comment:
> Operations are all little-endian

This does not sound right. The endianess matters only if you are reinterpreting bytes in memory as larger integers. None of these APIs do that.

Reply:
Could you please delete the "Operations are all little-endian." comment?

Reply:
> reinterpreting bytes in memory as larger integers

Ah. I never thought of it that way, fixed.